src/components/App/App.js

```diff
@@ -7,20 +7,45 @@ import SingleMovie from "../SingleMovie/SingleMovie";
 import HomeButton from "../HomeButton/HomeButton";
 
 function App() {
-  const mockData = movieData.movies;
-  const singleMockData = singleMovieData.movies;
-  const [movies, setMovies] = useState(mockData);
+  const [movies, setMovies] = useState([]);
   const [singleMovie, setSingleMovie] = useState(null);
+  const [error, setError] = useState(null);
 
-  function displaySingleMovie(id) {
-    const yourMovie = movies.find((movie) => {
-      return movie.id == id;
-    });
-    setSingleMovie(yourMovie);
-  }
+
+
+const getAllMovies = () => {
+  fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+  .then(response => response.json())
+  .then(data => {
+    console.log( "GET ALL MOVIES", data)
```

Comment:
I would encourage you to add a task to your project board to cleanup any remaining console.logs from your project!